serverless-test-project/rule-handler.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports.handleRule = (event, context, callback) => {
+  console.log(event)
```

Comment:
When the `event` arrives, it will be in the format of an EventBridge event, so it cannot have properties like `sleepSeconds` or `triggerError`. Instead, these would be part of the event `detail`.
The structure is documented here: https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents.html